src/Components/Fellowship/Fellowship.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const Fellowship = ({team, removePlayer}) => {
+
+  let teamMembers;
+
+  team.length ? 
```

Comment:
Took me a minute to figure out this was one big ternary. 
I wonder if an if/else would actually be easier to ready here? 
Or maybe just a formatting adjustment so it's a little clearer how they're connected.